### ISSUE TITLE
exp/clients/horizon: Add endpoint that exposes stellar core time

### ISSUE THREAD

**Issue by morleyzhi** [stellar/go#1008]
See https://github.com/stellar/js-stellar-sdk/pull/249.

Add a GET endpoint that returns the current time on stellar core, as a UNIX epoch (in seconds) expressed as a number. A sample response could be:

```json
{
  time: 1552594134
}
```

Once that's up, we can then change `stellar-sdk` to use this endpoint as the data source instead of Horizon's response headers.

**Comment by bartekn** [comment]
This can be added to root resource (`/`).

**Comment by sgehrman** [comment]
Is there an API in core to get it's time?  I took a quick look and didn't find it.

**Comment by bartekn** [comment]
> Would the time need to be added to Info?

Exactly, it's still not there. We haven't decided yet how important it is to have exact time in the context of stellar/js-stellar-sdk#249 so let's discuss it here first before bugging stellar-core devs 😉.

**Comment by sgehrman** [comment]
OK, I was looking for an issue to fix.  I can add it to core and horizon and do a PR.  Should I do this later?  Thanks

**Comment by bartekn** [comment]
Please hold on. We still need to discuss if it's worth doing this.
